clients/src/main/java/org/apache/kafka/common/metrics/stats/Rate.java

```diff
@@ -115,24 +114,10 @@ private double convert(long timeMs) {
         }
     }
 
-    public static class SampledTotal extends SampledStat {
-
-        public SampledTotal() {
-            super(0.0d);
-        }
-
-        @Override
-        protected void update(Sample sample, MetricConfig config, double value, long timeMs) {
-            sample.value += value;
-        }
-
-        @Override
-        public double combine(List<Sample> samples, MetricConfig config, long now) {
-            double total = 0.0;
-            for (Sample sample : samples)
-                total += sample.value;
-            return total;
-        }
-
+    /**
+     * @deprecated since 2.4 Use {@link SampledTotal} instead.
```

Comment:
The javadoc seems incorrect.

Reply:
Oops! I wonder how that happened... Thanks for catching it.